vector/src/androidTest/java/im/vector/app/EspressoExt.kt

```diff
@@ -70,6 +73,8 @@ fun waitForView(viewMatcher: Matcher<View>, timeout: Long = 10_000, waitForDispl
             val endTime = startTime + timeout
             val visibleMatcher = isDisplayed()
 
+            uiController.loopMainThreadForAtLeast(100)
```

Comment:
I really wanted to avoid this but this single change made the biggest difference for me when it came to reliability. 

Dialogs and bottomsheets would be `visible` but not interactable, back actions or button presses would be ignored half the time~.